presets/massa-station-preset.js

```diff
@@ -87,7 +89,7 @@ module.exports = {
           fontSize: "16px",
           fontWeight: "600",
           fontFamily: theme("fontFamily.Urbane"),
-          lineHeight: "19px",
+          // lineHeight: "19px",
```

Comment:
Is that intended!? 🤔 

If yes, remove the code then. There's no need to keep commented code on the codebase. Except I have no context about it!

Reply:
yes the comment is relative to the line height displacement issue on buttons, I commented to keep traceability and avoid future problems if revert is needed

Reply:
No need! Remove it if is not needed! BTW, are we sure about removing that!?

Reply:
we have two choices : 
change value of line height to fit to tailwind standard and manually replace with tail wind equivalent of : 
`position : relative;
top : -18px`

OR 

delete it

Reply:
some questions because I have no context about that: 

1) why 19px was set there!? 
2) is that still needed!?
3) if we remove that, it will make our life easier!?
4) what the impact of removing it!? Are we using in somewhere!?

Maybe @0xMazout can help with that...

Reply:
1. 19 px is set there because in lucas design system there is a line height of 19px 
<img width="235" alt="Screenshot 2023-05-03 at 17 18 52" src="https://user-images.githubusercontent.com/90157528/235961197-923c85c9-c6e2-41f9-84aa-e9422cde339a.png">

2.Yes he needs it in the design, but in this specific mas-menu-active preset we might not need it 
3. Yes ! 
4. Logically each time we use it we are in an active menu button (on the home page) so we will potentially face the same problem, which makes it logic to delete it  
<img width="358" alt="Screenshot 2023-05-03 at 17 20 40" src="https://user-images.githubusercontent.com/90157528/235961776-20d1dcec-c88f-461f-8423-952f6833596f.png">

Reply:
Ok! Got it! If we need to override that always, then fine to remove it! Let's wait for Lucas (dev) to confirm.

Reply:
As i understand , we have a problem on the value. Because 19px is not in the tailwind config

**If the value doesn't fit lets change the value and keep the line height in the preset.**

We define typo in preset so we just have to call it when it's needed. 

Lets ask to @LucasF243 if we can change this to 18 to fit tailwind standard. 

I think we should keep the line height value in the preset it's part of the typo.

Or if we delete here we delete everywhere to be iso on all typos

Reply:
Synced with Lucas, he will change the line height to 20px which is a standard tailwind :
`w-5`
So I will update the preset line height to be equal to 20px

Reply:
the change has not been implemented and I have not changed the line height preset in this mr